server/routes/auth.js

```diff
@@ -64,6 +70,53 @@ router.post("/register", validationMiddleware, async function (req, res, next) {
     }
 });
 
+router.get("/user", loginRequired, async function (req, res, next) {
+    try {
+        const { id } = req.user;
+        const user = await userController.findOneWithId(id);
+        if (!user) {
+            res.status(400).json({ errors: ["Please sign in"] });
+            return;
+        }
+        // create and return jwt with user obj
```

Comment:
I think we want to leave in the check for isChef in this route that it had before

Reply:
fixed!